### DISCUSSION TITLE
Is it possible to choose the trivy-operator scan frequency ?

### DISCUSSION THREAD

**Discussion by alexiiiiiiiiiiiiiii** [aquasecurity/trivy-operator#2042] [Help & Support]
I'm actually running trivy-operator on my K8s cluster with helm, is it possible to configure the scan frequency ?

**Comment by chen-keinan** [comment]
@alexiiiiiiiiiiiiiii No, its not possible, trivy operator follow the operator pattern, it reconcile resources based on events (create, update, delete).

**Reply by alexiiiiiiiiiiiiiii** [reply]
The problem is : I think trivy breaks my docker rate limit by pulling alpine images

**Reply by chen-keinan** [reply]
you can try to use [file system scan ](https://aquasecurity.github.io/trivy-operator/latest/tutorials/private-registries/) or add secret for docker hub with higher rate limit
